### ISSUE TITLE
Still having an issue?

### ISSUE THREAD

**Issue by coryhouse** [mafideju/ReactHouse#1]
I got a notification that you posted on the Redux course discussion, but I can't see it. If you're having an issue, please reply in this issue.

**Comment by snychka** [comment]
Dunno why, but it also didn't show up for me @coryhouse .

It wasn't deleted as of this post, so assuming they're still having a problem:

https://app.pluralsight.com/library/courses/react-redux-react-router-es6/discussion#comment-4550232291

**Comment by coryhouse** [comment]
The onSubmit in courseForm isn't set to anything. handleSubmit isn't being passed down on props from the parent. You're passing down onSave from EditCourses instead. I suggest comparing your code to the final course exercise (available under exercise files)

**Comment by mafideju** [comment]
Yes. Lots of hours wasted. And that feeling "i want my money back". After 3 months this is what to think about PluralSight and these worthless courses.

**Comment by coryhouse** [comment]
@mafideju - I'm sorry to hear that. Which course(s) frustrated you? What did you find frustrating or disappointing about the courses you've viewed?
